CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 ## 4.70.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_compute_network_endpoints` ([#14869](https://github.com/hashicorp/terraform-provider-google/pull/14869))
+* **New Resource:** `vertex_ai_index_endpoint` ([#14842](https://github.com/hashicorp/terraform-provider-google/pull/14842))
+
+IMPROVEMENTS:
+* bigtable: added timeout support for bigtable admin operations ([#14861](https://github.com/hashicorp/terraform-provider-google/pull/14861))
+* cloudfunctions2: added `url` output field to `google_cloudfunctions2_function` ([#14851](https://github.com/hashicorp/terraform-provider-google/pull/14851))
+* compute: added field `network_attachment` to `google_compute_instance_template` ([#14874](https://github.com/hashicorp/terraform-provider-google/pull/14874))
+* compute: surface error details for Compute Operation with quota exceeded errors. ([#14879](https://github.com/hashicorp/terraform-provider-google/pull/14879))
```

Comment:
This would be good to rewrite to be more focused on the customer experience - that is, not referring to API objects like operations.

Reply:
Looks like this changed `computed_operation`, I'm not sure exactly which operations does it refer here.

Reply:
Resolving this thread since it's outdated & not showing up on the changed view any more; I'll open a new thread with a suggestion.